db/migrations/schema-objects/20220901154203.do.check.startedat.sql

```diff
@@ -0,0 +1,5 @@
+
+ALTER TABLE
+  mtc_admin.[check]
+ADD
+  startedAt datetimeoffset(3) NULL
```

Comment:
This new column should really be documented in MSSQL too.

See: https://github.com/jon-shipley/patterns/blob/master/mssql-idempotent-sql.md#Documentation